src/tt.cpp

```diff
@@ -48,6 +47,8 @@ void TranspositionTable::resize(size_t mbSize) {
                 << "MB for transposition table." << std::endl;
       exit(EXIT_FAILURE);
   }
+  memset(mem, 0, clusterCount * sizeof(Cluster) + CacheLineSize - 1);
```

Comment:
Sorry, I meant clear() which is already a member of TranspositionTable so u don't have to add another manual memset() call.  Just call clear() at the very bottom of resize() after the table=... line.